From: Michael Olbrich <m.olbrich@pengutronix.de>
Date: Mon, 28 Nov 2016 10:26:54 +0100
Subject: [PATCH] setfiles: fix build error

Otherwise building fails with:

[...]
selinux/setfiles.c:150:22: error: 'bb_common_bufsiz1' undeclared (first use in this function)
[...]

Signed-off-by: Michael Olbrich <m.olbrich@pengutronix.de>
---
 selinux/setfiles.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/selinux/setfiles.c b/selinux/setfiles.c
index 51a7e63bd741..d8083d4fd580 100644
--- a/selinux/setfiles.c
+++ b/selinux/setfiles.c
@@ -46,6 +46,7 @@
 //usage:     "\n		if it has changed"
 
 #include "libbb.h"
+#include "common_bufsiz.h"
 #if ENABLE_FEATURE_SETFILES_CHECK_OPTION
 #include <sepol/sepol.h>
 #endif
